src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -1121,6 +1132,7 @@ class PersonAgent(
         } else {
           false
         }
+        val isSharedVehicle = vehicle.isSharedVehicle
```

Comment:
Why do you need this variable? Could you just use `vehicle.isSharedVehicle` as they use it below?